### DISCUSSION TITLE
[Discussions] Make `_` a news category

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3484] [Ideas]
Imo should the `_` not be a open category as it would allow anyone here to just spam posts, or create ones matching blog post titles, which in the end may or may not break stuff...

Just an idea.

**Comment by squidfunk** [comment]
Does this work with giscus? It’s important that comments can be posted by everyone without the need for me to create the discussion before.

**Reply by Andre601** [reply]
Everyone can comment on a discussion, even those in news categories. Giscus even recommends using that:
![image](https://user-images.githubusercontent.com/11576465/150549679-0a25d725-819e-4153-8c37-b6263accc6ae.png)

**Reply by squidfunk** [reply]
Perfect, thanks for noting! I'll change it straight away.
